File: 3264_final_array_state_after_k_multiplication_operations_i.py
class Solution:
    def getFinalState(self, nums: list[int], k: int, multiplier: int) -> list[int]:
        operations = 0
        while operations < k:
            min_value = min(nums)
            i = nums.index(min_value)
            nums[i] = nums[i] * multiplier
            operations += 1
        return nums

File: JavaScript/3264_final_array_state_after_k_multiplication_operations_i.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @param {number} multiplier
 * @return {number[]}
 */
var getFinalState = function(nums, k, multiplier) {
    for (let x = 0; x < k; x++) {
        const min = Math.min(...nums);
        nums[nums.indexOf(min)] *= multiplier;
    }
    return nums;
};

